plugins/+ciplugins/+github/BuildSummaryPlugin.m

```diff
@@ -2,27 +2,46 @@
 
 %   Copyright 2024 The MathWorks, Inc.
 
-    methods (Access=protected)
+    properties
```

Comment:
Seems like this property could be private

Reply:
I'm talking about the `TaskDetails` property here, not the methods.

Reply:
Oops my bad, got confused with the GitHub change view. Updating it!